### PR TITLE
Added a new CIP candidate

### PR DESCRIPTION
**Pull request by jjos2372**

### CODE REVIEW

File: cip-00XX.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    Status: Draft
+    Type: Hard Fork
+    Created: 2019-04-07
+
+
+## Motivation
+
+With Burst, smart contracts are implemented using Automated Transactions (AT), a technology created by the CIYAM developers. Automated Transactions are turing-complete and thus have a potentially infinite number of use cases. Beside this possible infinite number of use cases the current fee structure for AT on Burst-chain is counterproductive.
+To attract business and developers for an investment/involvement with the ATs on the Burstcoin blockchain the fee structure needs to be reviewed and changed.
+
```
Comment (frankTheTank72) [inline cip-00XX.md]: BURST is shipped with a smart contract implemented as Automated Transactions (AT). This AT system was developed by [CIYAM](http://ciyam.org/at/at_api.htm) and is turing-complete, thus having potentially infinite number of use cases. The concept of this virtual machine is time-tested and already proven on-chain with some complex examples; like the BURST-lottery: which was written with only 37 steps in CIYAM bytecode.

Beside this possible infinite number of use cases the current fee structure for AT on Burst-chain is counterproductive. To attract business and developers for an investment/involvement with the ATs on the Burstcoin blockchain the fee structure needs to be reviewed and changed.

File: cip-00XX.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+The parameters for the AT fee structure are handled in the AT_Constants.java file of Burst Reference Software (BRS). We propose to update the parameters in the following way (constants not changed are not listed):
+
+
+| Constant | AT version1 | AT version 2|
+|----------|-------------|-------------|
+| STEP_FEE |  0.1 BURST  | FEE_QUANT/10|
+| MAX_STEPS| 2000        | 100000      |
+| COST_PER_PAGE| 1 BURST | 10*FEE_QUANT|
+
```
Comment (frankTheTank72) [inline cip-00XX.md]: This constants in regards to AT will stay unchanged:


Constants | AT version 1 & 2
-- | --
MIN_FEE | 0.00001 BURST
API_STEP_MULTIPLIER | 10

File: cip-00XX.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+### Explanation of Parameters
+
```
Comment (frankTheTank72) [inline cip-00XX.md]: MIN_FEE
Currently unused, this constant is not referenced in BRS code. However, there is already a fee for every possible AT actions.

File: cip-00XX.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+**STEP_FEE** : This is the fee charged on every simple operation, like simple additions or subtractions.
+
+**MAX_STEPS** : The maximum number of steps (operations, steps, API calls) to be included in a block. When this number is reached the current state is stored and the execution continues in the next block.
+
```
Comment (frankTheTank72) [inline cip-00XX.md]: API_STEP_MULTIPLIER
BURST AT specification expose to AT programs several functions that need to consult the blockchain (e.g. address of a given transaction, amount in transaction, etc.). These functions are considered more costly to nodes and then each API call is actually considered as multiple steps. Effectively, the cost for these so called API calls is given by STEP_FEE*API_STEP_MULTIPLIER.

File: cip-00XX.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+1. COST_PER_PAGE: this cost is being reduced to a FEE_QUANT*10 == 0.0735 per page (256 bytes) when a new AT is registered (or created). Large AT programs can take several pages taking then more space in the blockchain and then affecting nodes. Even with the proposed modification each page (256 bytes) cost more than a message transaction can cost. Further, a message transaction can contain much more than 256 bytes (up to 4x times more). Under another perspective, registering a large (complex) AT mean potential for a large number of steps. This is, however, already taken care of by the step fees.
+
+
+### Compatibility
+
+The change will require a hard-fork. Any ATs created before the fork will be subject to the previous set of fees and will operate without any change. Only ATs created after the fork will run under the new fees.
+
```
Comment (frankTheTank72) [inline cip-00XX.md]: **Summary**
Unlike other blockchains, BURST does not have its own colored coin for running ATs. Registering and running ATs is supported by fees and the AT balance in BURST. With the current fees for ATs (version 1) any substantial increase in BURST price can make AT applications infeasible. In this CIP a new fee table is proposed (version 2). The proposed change would take place as a hard-fork and possibly attract new developers and applications for Burstcoin.

File: cip-00XX.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+## References
+
+BRS version example with new  setup :https://github.com/jjos2372/burstcoin/commit/53ba6c6dea775371c017256529e16289e808257b
+
```
Comment (frankTheTank72) [inline cip-00XX.md]: [CIP3](https://github.com/burst-apps-team/CIPs/blob/master/cip-0003.md)

File: cip-00XX.md (hunk 1 above)
Comment (frankTheTank72) [reply cip-00XX.md]: This text should remove the current Motivation text

File: cip-00XX.md (hunk 2 above)
Comment (frankTheTank72) [reply cip-00XX.md]: was missing

File: cip-00XX.md (hunk 5 above)
Comment (frankTheTank72) [reply cip-00XX.md]: section was missing

File: cip-00XX.md (hunk 2 above)
Comment (harry1453) [reply cip-00XX.md]: Missing parameters should not be included.

File: cip-00XX.md (hunk 3 above)
Comment (harry1453) [reply cip-00XX.md]: This will be removed separately then.

**Comment by harry1453** [comment]
OK for merge as inactive draft.

**Pull request merged**